proposals/lip-0048.md

```diff
@@ -91,7 +90,8 @@ def beforeCommandExecute(trs: Transaction) -> None:
     senderAddress = SHA256(trs.senderPublicKey)[:LENGTH_ADDRESS]
     generatorAddress = b.header.generatorAddress
 
-    if Token.getChainID(TOKEN_ID_FEE) == CHAIN_ID_ALIAS_NATIVE: # Fee token is a native token
+    let chainID be the chain ID of the current chain
+    if Token.getChainID(TOKEN_ID_FEE) == chainID: # Fee token is a native token
```

Comment:
probably we should have an API in the token module like `isNativeToken(tokenID) -> bool` to encapsulate this logic.